flake.nix

```diff
@@ -2,40 +2,42 @@
   description = "Flake for Holochain HTTP Gateway";
 
   inputs = {
-    nixpkgs.url = "nixpkgs/nixos-unstable";
+    holonix.url = "github:holochain/holonix?ref=main-0.4";
 
-    flake-parts.url = "github:hercules-ci/flake-parts";
-
-    rust-overlay = {
-      url = "github:oxalica/rust-overlay";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
+    nixpkgs.follows = "holonix/nixpkgs";
+    flake-parts.follows = "holonix/flake-parts";
+    rust-overlay.follows = "holonix/rust-overlay";
   };
 
-  outputs = { ... }@inputs:
-    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = [ "aarch64-darwin" "x86_64-linux" ];
+  outputs = inputs@{ flake-parts, ... }: flake-parts.lib.mkFlake { inherit inputs; } {
+    systems = builtins.attrNames inputs.holonix.devShells;
+    perSystem = { inputs', pkgs, system, ... }: {
+      _module.args.pkgs = import inputs.nixpkgs {
+        inherit system;
+        overlays = [ inputs.holonix.inputs.rust-overlay.overlays.default ];
+      };
 
-      perSystem = { pkgs, system, ... }: {
-        _module.args.pkgs = import inputs.nixpkgs {
-          inherit system;
-          overlays = [ inputs.rust-overlay.overlays.default ];
+      formatter = pkgs.nixpkgs-fmt;
+
+      devShells.default =
+        let
+          rustFromFile = pkgs.rust-bin.fromRustupToolchainFile ./rust-toolchain.toml;
+        in
+        pkgs.mkShell {
+          packages = (with inputs'.holonix.packages; [
```

Comment:
I think we need `hc` here, right? The fixture script calls `hc app` and `hc dna`

Reply:
That's included with `holochain`, we don't export the binaries from the Holochain repository one by one at the moment in Holonix. It's all one Nix package with several binaries